benchmark/benchmarks.jl

```diff
@@ -30,9 +30,9 @@ end samples = 3 evals = 1 seconds = 86400
 energy_problem = EnergyProblem(connection)
 
 SUITE["energy_problem"]["create_model"] = @benchmarkable begin
-    create_model!($energy_problem)
+    create_model!($connection, $energy_problem)
```

Comment:
We have the connection inside the `energy_problem` (named `db_connection`, maybe we can rename), so this can be avoided

Reply:
Yes, we can! it is done in the latest changes